src/Storages/MergeTree/MergeTreeIndexText.h

```diff
@@ -62,7 +61,9 @@ namespace DB
   * - Information about posting lists for each token:
   *    1. Header of posting list (VarUInt) (see PostingsSerialization::Flags).
   *    2. Cardinality of token (VarUInt).
-  *    3. Offset in file to the posting list (VarUInt) or embedded serialized posting list if EmbeddedPostings flag is set.
+  *    3. a) If EmbeddedPostings flag is set, posting list embedded into the dictionary block.
```

Comment:
Minor: L. 58 should say `Currently only raw and front-coded string formats are supported`.